lib/src/iterable.dart

```diff
@@ -1005,3 +1005,18 @@ extension IterableX<E> on Iterable<E> {
     return [t, f];
   }
 }
+
+extension IterableFutureX<E> on Iterable<Future<E>> {
+  /// Create a stream from a group of futures.
+  ///
+  /// The stream reports the results of the futures on the stream in the order
+  /// in which the futures complete.
+  /// Each future provides either a data event or an error event,
+  /// depending on how the future completes.
+  ///
+  /// If some futures have already completed when `Stream.fromFutures` is called,
+  /// their results will be emitted in some unspecified order.
+  ///
+  /// When all futures have completed, the stream is closed.
+  Stream<E> asStreamAwaited() => Stream.fromFutures(this);
```

Comment:
Shouldn't it just be called `.asStream()`?

Reply:
I thought about it, but it will get in conflict with the previous method behavior. What if the user don't want it to be awaited and just need as tream of Futures? We can discuss about that but it's hard to predict what will be the commom case.